src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -593,10 +635,16 @@ MulticopterPositionControl::parameters_update(bool force)
 		_params.hold_max_xy = math::max(0.f, v);
 		param_get(_params_handles.hold_max_z, &v);
 		_params.hold_max_z = math::max(0.f, v);
-		param_get(_params_handles.acc_up_max, &v);
-		_params.acc_up_max = v;
-		param_get(_params_handles.acc_down_max, &v);
-		_params.acc_down_max = v;
+		param_get(_params_handles.rc_flt_smp_rate, &(_params.rc_flt_smp_rate));
```

Comment:
I see why you didn't use block param here. How about we move the constraints into the main code where the params are being used?